{{cookiecutter.ioc_name}}/activate_env.sh

```diff
@@ -1,6 +1,34 @@
 #!/bin/bash
 
+if [ -f "/reg/g/pcds/pyps/conda/pcds_conda" ]; then
+  echo "* Sourcing pcds_conda to get access to conda"
+  # shellcheck disable=SC1091
+  source /reg/g/pcds/pyps/conda/pcds_conda
+fi
+
+if [ ! -f "config.sh" ]; then
+  echo "* IOC configuration not found!"
+  exit 1
+fi
+
+echo "* Sourcing this IOC's configuration"
 . config.sh
 
-conda deactivate
-conda activate "$CONDA_ENV_PATH"
+echo "  - IOC:                     $IOC"
+echo "  - IOC_COMMAND:             $IOC_COMMAND"
+echo "  - PREFIX:                  $PREFIX"
+echo "  - IOC_DATA_PATH:           $IOC_DATA_PATH"
+echo "  - CONDA_ENV_PATH:          $CONDA_ENV_PATH"
+echo "  - EPICS_CA_AUTO_ADDR_LIST: $EPICS_CA_AUTO_ADDR_LIST"
+echo "  - EPICS_CA_ADDR_LIST:      $EPICS_CA_ADDR_LIST"
+echo "  - CAPTURE_LOGS:            $CAPTURE_LOGS"
+echo "  - LOG_FILE_PATH:           $LOG_FILE_PATH"
+
+echo "* Deactivating any existing conda environment"
+conda deactivate || echo "Conda deactivate failed; continuing..."
+
+echo "* Activating the conda environment: $CONDA_ENV_PATH"
+conda activate --no-stack "$CONDA_ENV_PATH"
```

Comment:
I did not know environment stacking was a thing until I saw this.  This seems safe but now I'm paranoid that it's enabled somewhere without my knowledge.

Reply:
This is probably an unnecessary change.

The stacking behavior is sensible and very weird at the same time... And when I was looking into the issue I thought "well, we definitely don't want a stack here..." 🤷‍♂️

Reply:
I had no idea there was a way to sidestep the stacking